trie/iterator.go

```diff
@@ -17,25 +17,72 @@
 package trie
 
 import (
-	log "github.com/inconshreveable/log15"
+	"fmt"
+	//log "github.com/inconshreveable/log15"
```

Comment:
Why not use log15?

Reply:
i'm not sure how to format things properly with log15.. i need stuff to be printed in hex and binary formats